main/src/ca/uwaterloo/flix/language/ast/NamedAst.scala

```diff
@@ -118,7 +118,7 @@ object NamedAst {
 
     case class Stm(exp1: NamedAst.Expression, exp2: NamedAst.Expression, loc: SourceLocation) extends NamedAst.Expression
 
-    case class Let(sym: Symbol.VarSym, exp1: NamedAst.Expression, exp2: NamedAst.Expression, loc: SourceLocation) extends NamedAst.Expression
+    case class Let(mod: Ast.Modifiers, sym: Symbol.VarSym, exp1: NamedAst.Expression, exp2: NamedAst.Expression, loc: SourceLocation) extends NamedAst.Expression
```

Comment:
Sorry to be difficult (again), but I really think `sym` should always be the first argument, because it is so important, so:

```
case class Let(sym: Symbol.VarSym, mod: Ast.Modifiers, exp1: NamedAst.Expression, exp2: NamedAst.Expression, loc: SourceLocation)
```

Reply:
I'm very proud of my vim substitution that fixed this.